monai/networks/nets/milmodel.py

```diff
@@ -0,0 +1,222 @@
+from enum import Enum
+from typing import Callable, Dict, List, Optional, Sequence, Union
+
+import torch
+import torch.nn as nn
+import torchvision
```

Comment:
Please import `torchvision` as optional like here:
https://github.com/Project-MONAI/MONAI/blob/8c0ff376587797aabd2abbdb569fb691880fc094/monai/networks/nets/torchvision_fc.py#L17

Reply:
done

Reply:
The same things apply to the tests. Please change 
```python
from torchvision.models import densenet121
```
thanks.